### ISSUE TITLE
Getting an error

### ISSUE THREAD

**Issue by decker** [mroach/NLog.RollbarSharp#4]
Seems package bit outdated.

[MissingMethodException: Method not found: 'Void RollbarSharp.RollbarClient.Send(RollbarSharp.Serialization.DataModel)'.]
   NLog.RollbarSharp.RollbarTarget.Write(LogEventInfo logEvent) +0
   NLog.Targets.Target.Write(AsyncLogEventInfo logEvent) +58

Send function accepts two parameters now.

**Comment by LucasMetal** [comment]
Yeah, I run into this today, I needed to use my own version of the target.
I'll try to send a PR with a fix.

Is this project still alive anyway?
